### ISSUE TITLE
MSI cannot be downloaded

### ISSUE THREAD

**Issue by pebernath** [maurice-daly/DriverAutomationTool#524]
The new MSI file (version 7.2.3) cannot be downloaded because the Microsoft defender warns it contains a virus.
I think it should be a false positive as the .exe file can be downloaded without any issue on same machine.

![virus found](https://github.com/maurice-daly/DriverAutomationTool/assets/133517554/5d888875-0694-4a38-a1c6-284b925bcf42)

**Comment by pebernath** [comment]
https://www.virustotal.com/gui/file/6b3e8a777bbc567b4be33be593d563109ce9ec205ba9a5864f90d6e4ad986b1b/detection

**Comment by nodiaque** [comment]
The installed one also triggered on my trend micro, I just lost the file after it's installed.

**Comment by ericvandam** [comment]
+1

**Comment by R4PH1** [comment]
The .ps1 file is also missing again. Could this maybe the added to the 7.2.3 folder additionally to the .exe?

**Comment by nodiaque** [comment]
the ps1 is also infected from 7.2.3

**Comment by ajn142attamu** [comment]
If anybody needs to grab 23H2 stuff until the .msi gets sorted out, I've edited my fork to add initial support for 23H2. You can copy https://github.com/ajn142attamu/DriverAutomationTool/blob/master/Current%20Branch/7.2.2/DriverAutomationTool.ps1 into `C:\Program Files\MSEndpointMgr\Driver Automation Tool` and then run it as admin. You may need to clear the `C:\Program Files\MSEndpointMgr\Driver Automation Tool\Temp` directory and rename `C:\Program Files\MSEndpointMgr\Driver Automation Tool\Settings\OEMLinks.xml` to ensure you pull the latest copy of OEMLinks.xml and then the latest copy of all the catalog files.

**Comment by Shirkan79** [comment]
+1

**Comment by xirv** [comment]
+1

**Comment by ajn142attamu** [comment]
> +1
> 
> Anyone have the MSI from 7.2.2?

Here’s a link to 7.2.2 from the repo commit history.

https://github.com/maurice-daly/DriverAutomationTool/blob/595768f9067095c499f031a5851655af124ff024/Driver%20Automation%20Tool.msi

**Comment by ajn142attamu** [comment]
> Your version work perfectly! I'm nearly wondering if I should just take the ps1 and recreate the projet files. Also add some threading so the gui stop freezing. Would be alot of work though

I’m not crazy about doing a lot of work if Maurice is going to take a more open approach with v8.x in the next 2 months, because I can’t guarantee I’ll be maintaining my fork forever. I’d say address that if/when it supports something else. I don’t have licensing for Sapiens anyways, just doing this all from VS Code.

**Comment by nodiaque** [comment]
Yeah I'm waiting to see v8. I do have sapiens powershell and honestly, since the ps1 is already all shared, I don't see why not sharing the OEM files (or including the recovery info which sapiens does per default). I can manually reconstruct from the PS1 itself manually, long job but not hard. But I'm willing to wait to see the futur. I understand the fact he invested a lot of time and effort, but he won't be able to maintain it forever. Opensource is the way to go with stuff like this.
